Formula/s3cmd.rb

```diff
@@ -30,13 +30,13 @@ class S3cmd < Formula
   end
 
   resource "python-magic" do
-    url "https://files.pythonhosted.org/packages/84/30/80932401906eaf787f2e9bd86dc458f1d2e75b064b4c187341f29516945c/python-magic-0.4.15.tar.gz"
-    sha256 "f3765c0f582d2dfc72c15f3b5a82aecfae9498bd29ca840d72f37d7bd38bfcd5"
+    url "https://files.pythonhosted.org/packages/26/60/6d45e0e7043f5a7bf15238ca451256a78d3c5fe02cd372f0ed6d888a16d5/python-magic-0.4.22.tar.gz"
+    sha256 "ca884349f2c92ce830e3f498c5b7c7051fe2942c3ee4332f65213b8ebff15a62"
   end
 
   resource "six" do
```

Comment:
Can this use the formula?

Reply:
I don't think so. `s3cmd` depends on `python@3.8`, `six` depends on `python@3.9`

Reply:
We can probs make `six` depend on `python@3.9` for the build and test. Been contemplating that since it would make it useful for another formula too. Would that work here?

Reply:
With some adjustments, I guess so. 
`six` installs python package into`lib/python3.9/site-packages/`, so we'll need to make it work for `python@3.8` (and maybe for `python@3.7` as well). And now I'm not sure how to keep track of all python versions we have.
I'm not against this idea, but I'd rather not mixing anything from different python versions :)

Reply:
Oof, right, ok. Does this code change between minor Python versions? I'd guess not. If so, maybe we can just get away with symlinking `lib/python3.8` to `lib/python3.9`, etc.

Reply:
Does this help? #77513

Reply:
Yep, it works, thanks!
Will update the PR